src/WebJobs.Script/Rpc/FunctionRegistration/FunctionDispatcher.cs

```diff
@@ -246,7 +246,16 @@ public async void WorkerError(WorkerErrorEvent workerError)
             if (!_disposing)
             {
                 _logger.LogDebug("Handling WorkerErrorEvent for runtime:{runtime}, workerId:{workerId}", workerError.Language, workerError.WorkerId);
-                _languageWorkerErrors.Add(workerError.Exception);
+
+                if (workerError.Exception == null)
```

Comment:
change this to something like
```
if (!ShouldIgnoreLanguageWorkerError(workerError.Exception))
                {
                    _languageWorkerErrors.Add(workerError.Exception);
                }

internal bool ShouldIgnoreLanguageWorkerError(Exception workerErrorEx)
        {
            if (workerErrorEx is LanguageWorkerProcessExitException processExitException)
            {
                if (processExitException.ExitCode == 200)
                {
                    _logger.LogDebug("No exception in WorkerErrorEvent for runtime:{runtime}, workerId:{workerId}", workerError.Language, workerError.WorkerId);
                    return true;
                }
            }
            return false;
        }
```
Adding Unit tests for ShouldIgnoreLanguageWorkerError would be enough.